packages/server/lib/modes/run.js

```diff
@@ -1412,7 +1423,12 @@ module.exports = {
               headed: options.headed,
               outputPath: options.outputPath,
             })
-            .tap(renderSummaryTable(runUrl))
+
+            if (config.quiet !== true) {
+              return runSpecs.tap(renderSummaryTable(runUrl))
+            }
+
+            return runSpecs
```

Comment:
instead of doing this, just add a conditional into the .tap:

```suggestion
			.tap((runSpecs) => {
			  if (!config.quiet) {
			  	renderSummaryTable(runUrl)(runSpecs)
			  }
            })
```